packages/rspack/package.json

```diff
@@ -19,7 +19,11 @@
   "generators": "./generators.json",
   "executors": "./executors.json",
   "dependencies": {
-    "@nrwl/devkit": "^15.7.2",
-    "copy-webpack-plugin": "^10.2.4"
+    "@nrwl/linter": "^15.8.1",
+    "@nrwl/devkit": "^15.8.1",
+    "ajv": "^8.12.0",
```

Comment:
There is a conflict with other packages, so make sure `ajv@8` is installed and matches what `@rspack/core` uses.